notebooks/abbash_notebook.ipynb

```diff
@@ -1,28 +1,540 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">for N in [20]:
&nbsp; &nbsp; step = (10) / (N+1) # create step size of (10 sec)/(20 samples)
&nbsp; &nbsp; t = np.arange(0,10+step, step)
&nbsp; &nbsp; func= np.sin(2*np.pi*t)
&nbsp; &nbsp; FFT= np.fft.fft(func)
&nbsp; &nbsp; freq_step = (N/time) / len(FFT)
&nbsp; &nbsp; freqs = np.arange(0,N/time, freq_step)
&nbsp; &nbsp; plt.plot(freqs, np.absolute(FFT))
</pre><p>In the text: "Here you use a step size of 10 seconds divided by 21 with <code>step = 10/(N+1)</code>"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>